tests/entrypoints/test_webapp.py

```diff
@@ -0,0 +1,44 @@
+import io
+import json
+from unittest.mock import Mock
+
+import pytest
+from flask import Flask
+
+from counter.entrypoints.webapp import create_app
+
+
+@pytest.fixture
+def client():
+    app = create_app()
+
+    app.config['TESTING'] = True
+
+    with app.test_client() as client:
+        yield client
+
+import os
+
+def test_object_detection(client):
+    
+    # Load the image from the path resource/boy.jpg
+    # Leo, please, check if there is a more elegant way of doing this (fixture?)
+    image_path = os.path.abspath(os.path.join(os.path.dirname(__file__), '..','..', 'resources','images','boy.jpg'))
```

Comment:
Use this snippet:
The first "parent" is to get the dirname, then the other two are for each ".." you have there.
```python
from pathlib import Path
...
@pytest.fixture
def image_path():
    ref_dir = Path(__file__).parent
    return ref_dir.parent.parent / "resources" / "images" / "boy.jpg"

def test_object_detection(client, image_path):
...
```